better-font-awesome.php

```diff
@@ -513,18 +546,52 @@ public function text_callback( $args ) {
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="fa-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="fa-2x"]</code> or <code>&lt;i class="fa-coffee fa-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="fa-2x fa-rotate-90"]</code> or <code>&lt;i class="fa-coffee fa-2x fa-rotate-90"&gt;&lt;/i&gt;</code><br /><br /><br />
-                     <b>Font Awesome version 3.x</b>&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/3.2.1/examples/">See all available options &raquo;</a></small><br /><br />
-                     <i class="icon-coffee fa fa-coffee"></i> <code>[icon name="coffee"]</code> or <code>&lt;i class="icon-coffee"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x"></i> <code>[icon name="coffee" class="icon-2x"]</code> or <code>&lt;i class="icon-coffee icon-2x"&gt;&lt;/i&gt;</code><br /><br />
-                     <i class="icon-coffee fa fa-coffee icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="coffee" class="icon-2x icon-rotate-90"]</code> or <code>&lt;i class="icon-coffee icon-2x icon-rotate-90"&gt;&lt;/i&gt;</code>',
-                'better-font-awesome' ) .
-                '</div>';
+        return '<div class="bfa-usage-text">' .
+
+                    '<h3>' . esc_html__( 'Usage', 'better-font-awesome' ) . '</h3>' .
+                    sprintf(
+	                    esc_html__(
+		                    'Font Awesome version 4.x + %s See all available options %s',
+		                    'better-font-awesome'
+	                    ),
+	                    '&nbsp;&nbsp;&nbsp;<small><a href="http://fontawesome.io/examples/">',
+	                    '&raquo;</a></small><br/><br/>'
+                    ) .
+                    sprintf(
+	                    '<i class="icon-%$1s fa fa-%$1s"></i> <code>[icon name="%$1s"]</code> %$2s <code>&lt;i class="fa-%$1s"&gt;&lt;/i&gt;</code><br /><br />' .
+	                    '<i class="icon-%$1s fa fa-%$1s icon-2x fa-2x"></i> <code>[icon name="%$1s" class="fa-2x"]</code> %$2s <code>&lt;i class="fa-%$1s fa-2x"&gt;&lt;/i&gt;</code><br /><br />' .
+	                    '<i class="icon-%$1s fa fa-%$1s icon-2x fa-2x icon-rotate-90 fa-rotate-90"></i> <code>[icon name="%$1s" class="fa-2x fa-rotate-90"]</code> %$2s <code>&lt;i class="fa-%$1s fa-2x fa-rotate-90"&gt;&lt;/i&gt;</code><br /><br /><br />',
+	                    esc_html__(
+		                    'coffee',
+		                    'better-font-awesome'
+	                    ),
+	                    esc_html__(
+		                    'or',
+		                    'better-font-awesome'
+	                    )
+                    ) .
```

Comment:
@tcrsavage aha, thanks for the example - now I see what you mean. Any pointers to read up on templating?